tests/e2e/configs/inversify.config.ts

```diff
@@ -43,37 +44,55 @@ import { OcpMainPage } from '../pageobjects/openshift/OcpMainPage';
 import { OcpImportFromGitPage } from '../pageobjects/openshift/OcpImportFromGitPage';
 import { OcpApplicationPage } from '../pageobjects/openshift/OcpApplicationPage';
 
-const e2eContainer: Container = new Container({defaultScope: 'Transient'});
+const e2eContainer: Container = new Container({ defaultScope: 'Transient' });
 
 e2eContainer.bind<IDriver>(TYPES.Driver).to(ChromeDriver).inSingletonScope();
-e2eContainer.bind<ITestWorkspaceUtil>(TYPES.WorkspaceUtil).to(TestWorkspaceUtil);
+e2eContainer
+  .bind<ITestWorkspaceUtil>(TYPES.WorkspaceUtil)
```

Comment:
Could you please explain, for what reason have you add this new lines?

Reply:
I didn't manually touch the file before and after linting it so this was done by the linter itself.